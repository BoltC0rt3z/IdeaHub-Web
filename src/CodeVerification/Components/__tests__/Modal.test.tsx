import React from 'react';
import { mount, ReactWrapper } from 'enzyme';

import VerificationModal from '../Modal';

describe('Tests for modal component', () => {
  const wrapper: ReactWrapper = mount(<VerificationModal />);

  it('Should render the modal without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
