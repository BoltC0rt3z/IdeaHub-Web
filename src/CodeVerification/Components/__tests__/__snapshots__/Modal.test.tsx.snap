// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for modal component Should render the modal without crushing 1`] = `ReactWrapper {}`;
